src/pages/profile/edit/profile-settings/index.tsx

```diff
@@ -11,30 +11,32 @@ import { LOCAL_STORAGE_KEYS } from '@src/config/local-storage.config';
 import { useAppSelector } from '@src/hooks/redux-hooks';
 import { getProfile } from '@src/stores/slices/authSlice';
 import { ThemeMode } from '@src/types/profile.types';
-import { getValueFromLocalStorage } from '@src/utils';
+import { convertCodeToLanguage, getValueFromLocalStorage } from '@src/utils';
+import { DEFAULT_LANGUAGE } from '../../constants';
 import EditablePanel from '../EditablePanel';
 import { ProfileCurrency } from './ProfileCurrency';
 import { ProfileLang } from './ProfileLang';
 
 function ProfileSettings() {
   const profileId = useAppSelector(getProfile)?.id;
+  const profile = useAppSelector(getProfile);
```

Comment:
you can combine getting `profile.language` and `profileId` with help of destruction assignment

```ts
const profile = useAppSelector(getProfile)
  const { language: userLang, id: profileId } = profile ?? {}
```